### PR TITLE
fix(radio): set box-sizing to border-box

### PR DESCRIPTION
**Pull request by Mohammer5**

### CODE REVIEW

File: src/Radio/styles.js [hunk 1]
```diff
@@ -16,6 +16,7 @@ export default css`
         height: calc(100% + 2px);
         top: -1px;
         left: -1px;
+        box-sizing: border-box;
```
Comment (varl) [inline src/Radio/styles.js]: Is this necessary when we load in the [`CSSReset`](https://github.com/dhis2/ui-core/blob/master/src/CssReset/index.js#L15)?

Comment (Mohammer5) [reply src/Radio/styles.js]: The import export app doesn't use the `CSSReset` and either we fix the component there or we make sure that the component can be used with and without the reset css

Comment (ismay) [reply src/Radio/styles.js]: Would adding the CssReset there break things? Might also be a solution?

Comment (Mohammer5) [reply src/Radio/styles.js]: That seems to work fine actually

**Comment by Mohammer5** [comment]
`CssReset` should be used if possible

**Pull request closed without merging**